easybuild/easyconfigs/a/ASE/ASE-3.22.0-intel-2020b.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.22.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Flask', '1.1.2'),
+    ('matplotlib', '3.3.3'),
+    ('Tkinter', '%(pyver)s'),     # Needed by GUI of ASE
+    ('spglib-python', '1.16.0'),  # optional
+]
+
+use_pip = True
+sanity_pip_check = True
+
+# required because we're building Python packages (MarkupSafe,
+# ase-ext) using Intel compilers on top of Python built with GCC.
+check_ldshared = True
```

Comment:
@schiotz This is done automatically now, since quite a while (since EasyBuild v4.0.0, see https://github.com/easybuilders/easybuild-easyblocks/pull/1788), so you can remove this...